metaworld/envs/mujoco/sawyer_xyz/base.py

```diff
@@ -110,6 +126,7 @@ def set_xyz_action(self, action):
         self.data.set_mocap_quat('mocap', np.array([1, 0, 1, 0]))
 
     def discretize_goal_space(self, goals):
+        assert False
```

Comment:
Should we keep these asserts? I was just checking if these methods were called at all.